settings.gradle

```diff
@@ -3,19 +3,12 @@ pluginManagement {
         maven {
             // to load the gradle-josm-plugin from a custom maven repo hosted
             // on in a custom S3 bucket
-            url "$gradle_josm_plugin_repo"
-        }
-        gradlePluginPortal()
-    }
-
-    resolutionStrategy {
-        eachPlugin {
-            // WORKAROUND: we have to use the plugin ID 'org.openstreetmap.josm'
-            // instead of 'gradle-josm-plugin', see comment at the beginning
-            // of 'build.gradle'.
-            if (requested.id.id == 'org.openstreetmap.josm') {
-                useModule("org.openstreetmap.josm:gradle-josm-plugin:$gradle_josm_plugin_version")
+            url = uri(gradle_josm_plugin_repo)
+            content {
+                includeModule("org.openstreetmap.josm", "gradle-josm-plugin")
+                includeModule("org.openstreetmap.josm", "org.openstreetmap.josm.gradle.plugin")
             }
```

Comment:
```suggestion
```
This `content{}` block is optional. It restricts Gradle, so that only these two artifacts will be downloaded from there.
For other artifacts this repository will no longer be considered.